planner/core/explain.go

```diff
@@ -696,8 +700,44 @@ func (p *TiKVSingleGather) ExplainInfo() string {
 
 // ExplainInfo implements Plan interface.
 func (p *PhysicalMemTable) ExplainInfo() string {
-	if !metricschema.IsMetricTable(p.Table.Name.L) {
+	return genMetricTableExplainInfo(p.ctx, p.DBName, p.Table, p.Extractor)
```

Comment:
I think there is unnecessary to define a new function, it's better to inline it.

Reply:
great.